client/asset/eth/eth.go

```diff
@@ -192,13 +202,10 @@ type ExchangeWallet struct {
 	tipMtx     sync.RWMutex
 	currentTip *types.Block
 
-	acct *accounts.Account
+	acct *lockedAcct
 
 	cachedInitGas    uint64
 	cachedInitGasMtx sync.Mutex
-
-	lockedFunds    map[string]uint64 // gwei
```

Comment:
We discussed having the map vs just an integer over here:
https://github.com/decred/dcrdex/pull/1221#discussion_r722888631

I don't think there's any benefit other than serving as a safety check, but I don't think there's any downside either.

Having the `lockedAcct` struct is definitely nicer though in case we decide to support multiple addresses.

Reply:
I recall that discussion, and the earlier one about adding the nonce in the first place.  https://github.com/decred/dcrdex/pull/1221#discussion_r722737273
I'm still not convinced the more complicated coin id encoding helps much, and I'm trying to get to the bottom of it in the PR, which is just for discussion at this point.

Reply:
Yeah I'm not opposed to removing this map either tbh, I think it might be more trouble than it's worth.